src/NuGet.Services.AzureSearch/SearchService/SearchParametersBuilder.cs

```diff
@@ -202,6 +204,12 @@ private static IList<string> GetOrderBy(V2SortBy sortBy)
                 case V2SortBy.CreatedDesc:
                     orderBy = CreatedDesc;
                     break;
+                case V2SortBy.TotalDownloadsAsc:
+                    orderBy = TotalDownloadsAsc;
+                    break;
+                case V2SortBy.TotalDownloadsDesc:
+                    orderBy = TotalDownloadsDesc;
+                    break;
```

Comment:
The `IndexOperationBuilder.V2SearchWithHijackIndex` should check that the request does not have `packageType` or `orderBy` parameters. We should cover this with unit tests and functional tests.